src/AuthModule/Components/ForgotPassword/ForgotPassword.module.css

```diff
@@ -0,0 +1,32 @@
+.loader {
```

Comment:
The load will probably be used across the application, so we shouldn't limit it to only ForgetPassword.
There is a loader implementation in MUI, we take the code and abstract it into a generic component.

loader implementation:
https://mui.com/material-ui/react-backdrop/